gpkit/feasibility.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+from .variables import Variable, VectorVariable
+from .substitution import get_constants
+from .varkey import VarKey
+from .posyarray import PosyArray
+
+
+def feasibility_model(model, flavour="max", varname=None, constants=None):
+    """Returns a new GP for the closest feasible point of the current GP.
```

Comment:
Hmm. It does, but it makes it harder to write automated tools...how do you feel about the extra attributes?